kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -420,15 +421,24 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
         s"The valid batch state can be one of the following: ${VALID_BATCH_STATES.mkString(",")}")
     }
 
+    val createTimeFilter =
+      math.max(createTime, metadataSearchWindow.map(System.currentTimeMillis() - _).getOrElse(0L))
     val filter = MetadataFilter(
       sessionType = SessionType.BATCH,
       engineType = batchType,
       username = batchUser,
       state = batchState,
       requestName = batchName,
-      createTime = createTime,
+      createTime = createTimeFilter,
       endTime = endTime)
```

Comment:
not related to this PR, but `createTime` and `endTime` are confusing, should be `minCreateTime` and `maxEndTime`, and say if they are inclusive or exclusive in the comments, with an explanation for specific values

Reply:
update: the predication name might be more intuitive if we use `greatThanXXX` `lessThanOrEqualsXXX`

Reply:
Will address it in another PR